### PR TITLE
[EDU-771] Render the DEMO cards in the tutorials page

### PR DESCRIPTION
**Pull request by fliptopbox**
Render the DEMO cards in the tutorials page

## Description

Create the textile files:
Here is the current list of all Ably Dev Rel Demos as a google sheet:

[Ably - List of Demos (used on tutorials page)](https://docs.google.com/spreadsheets/d/1M1Tenb02zZKsVTma6WpSQ3SfW0poIzp9UDo8KnsBPUE/edit#gid=0) (there are currently 15 items to convert)

These will be used to create the initial demo textile article, with a very short excerpt based on the sheets description column. I have created an example textile that we will use as template to create all the demo files. 

- Clone this branch[GitHub - ably/docs at add-demo-cards](https://github.com/ably/docs/tree/add-demo-cards) 
- Navigate to the content/tutorials folder, and find [demo-example-card.textile](https://github.com/ably/docs/blob/add-demo-cards/content/tutorials/demo-example-card.textile) 
- All the new demo files need to have a filename that starts with demo-*******.textile
- Open the template and save that as a new file, edit the file and replace the following fields with the values from the google sheet:
  - author, date_published, last_updated (iso date for today) excerpt & meta_description (are the same as the description in the google sheet) demo_url and demo_repo, and update languages and tags to represent the demo labels.
  - replace the body of the document with a H2 heading of the title and the description as the body

Every demo MUST contain `Demo` in the list of tags.If it NOT present the demo will **NOT** appear as a demo!

## Review

Instructions on how to review the PR. 

- Read this ticket https://ably.atlassian.net/browse/EDU-771
- Preview App: https://website-edu-771-add-dem-n3gg3p.herokuapp.com/
- To merge into production i require 1st the docs repo and 2nd the website repo, in that order. thanks.
- Website component PR: https://website-edu-771-add-dem-n3gg3p.herokuapp.com/
- Complete list of demos [on Google Sheets](https://docs.google.com/spreadsheets/d/1M1Tenb02zZKsVTma6WpSQ3SfW0poIzp9UDo8KnsBPUE/edit?usp=sharing)

### CODE REVIEW

**Comment by fliptopbox** [comment]
hi @tbedford not sure who to ask for a PR review for the update DEMO files.

**Comment by tbedford** [comment]
@fliptopbox - I won't have time to look at this any time soon I'm afraid. I'm sure @marklewin and @leggetter are familiar with this and could test locally etc. Let me know if that's not going to be possible, and I will try and shuffle things around. Thanks.

File: content/tutorials/demo-realtime-midi-player.textile [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  author_name: owen pearson
+  author_profile_url: 'https://github.com/owenpearson'
+category:
+- channels
+- reactor-integrations
+- push-notifications
+- api-hub
+date_published: '2020-08-20T09:48:18Z'
+last_updated: '2020-08-20T09:48:18Z'
+excerpt: A realtime midi keyboard in the browser. Uses WebAPIs and pubsub to create a virtual jam session.
```
Comment (marklewin) [inline content/tutorials/demo-realtime-midi-player.textile:20]:
```suggestion
excerpt: A realtime MIDI keyboard in the browser. Uses Web APIs and pub/sub to create a virtual jam session.
```

File: content/tutorials/demo-realtime-midi-player.textile [hunk 2]
```diff
@@ -0,0 +1,41 @@
+---
+ably_product: reactor
+platform: browser
+title: Realtime midi player
+meta_description: A realtime midi keyboard in the browser. Uses WebAPIs and pubsub to create a virtual jam session.
```
Comment (marklewin) [inline content/tutorials/demo-realtime-midi-player.textile:5]:
```suggestion
meta_description: A realtime MIDI keyboard in the browser. Uses Web APIs and pub/sub to create a virtual jam session.
```

File: content/tutorials/demo-realtime-multiplayer-space-invaders.textile [hunk 3]
```diff
@@ -0,0 +1,50 @@
+---
+ably_product: reactor
+platform: browser
+title: Multiplayer Space Invaders
+meta_description: "A Realtime Multiplayer Space Invaders Game with Phaser 3 and Ably
+
+This demo shows how to build a multiplayer game which runs in the browser with a central NodeJS server controlling the game state.
+
+The game is built using PhaserJS."
```
Comment (marklewin) [inline content/tutorials/demo-realtime-multiplayer-space-invaders.textile:9]:
```suggestion
meta_description: A realtime multiplayer Space Invaders game built with Ably and the Phaser 3 game framework. The game runs in the browser and its state is managed by a central NodeJS server.
```

File: content/tutorials/demo-realtime-multiplayer-space-invaders.textile [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+- reactor-integrations
+- push-notifications
+- api-hub
+date_published: '2020-05-21T22:43:39Z'
+last_updated: '2020-05-21T22:43:39Z'
+excerpt: "A Realtime Multiplayer Space Invaders Game with Phaser 3 and Ably
+
+This demo shows how to build a multiplayer game which runs in the browser with a central NodeJS server controlling the game state.
+
+The game is built using PhaserJS."
```
Comment (marklewin) [inline content/tutorials/demo-realtime-multiplayer-space-invaders.textile:28]:
```suggestion
excerpt: A realtime multiplayer Space Invaders game built with Ably and the Phaser 3 game framework. The game runs in the browser and its state is managed by a central NodeJS server.
```

**Review by marklewin — CHANGES_REQUESTED** [review]
Some of the card text doesn't read well, or even fit in the cards. I've suggested some alternative text for the excerpts.

Also, I note from the [demo app](https://website-edu-771-add-dem-n3gg3p.herokuapp.com/tutorials) that clicking on the card doesn't do anything, whereas the UI suggests that it should? Maybe it should link to the GH repo and then perhaps you could just have "Live demo" as a link at the bottom of each card?

**Comment by fliptopbox** [comment]
@marklewin please review. thanks

**Review by marklewin — COMMENTED** [review]
@fliptopbox I can't see if the new text fits the cards, because the build seems to have failed? The text changes look good however, although I note that the meta_description in the Google Sheet doesn't match what we're now using here.

**Comment by fliptopbox** [comment]
@marklewin 

1. i have added a new `excert` column to track the differences in the [google sheet](https://docs.google.com/spreadsheets/d/1M1Tenb02zZKsVTma6WpSQ3SfW0poIzp9UDo8KnsBPUE/edit?usp=sharing) with respective word count - i will need to update the repos as part of Git gardening sessions.
2. You can see the [docs preview app here](https://ably-docs-add-demo-card-mpja6b.herokuapp.com/), the UI changes **will not** be reflected on the website UI (that the preview app is for Website and uses SHA from the previous commit)

**Review by marklewin — APPROVED** [review]
Not able to preview, but text LGTM! 👍

**Review by marklewin — APPROVED** [review]
Re-approving, because @fliptopbox still unable to merge.

**Pull request merged**